README.md

```diff
@@ -1,57 +1,25 @@
 # Northcoders House of Games API
 
-## Background
+## Installing Necassary Files for Running
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+Before using this repository, a few files will need to be created in order for it to work. In order for the databases to be seeded properly, .env.test and env.development should be created and populated in the root directory of the project in order to set the ENV enviroment.
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+The contents of .env.test should be:
 
-## Kanban
+> PGDATABASE=nc_games_test
 
-### Link to your Trello Board here: https://trello.com/b/gBtX4GN3/be-nc-games
+The contents of .env.development should be:
 
-To keep track of the tasks involved in this project we're going to use a kanban board. Ensure that you work on one _ticket_ at time. You can click on the ticket to find out more information about what is required for the feature. A ticket is not considered complete unless both the happy path and errors response are handled. You can make use of the checklist on each ticket to keep track of the errors you want to handle. You can also make use of [error-handling.md](error-handling.md) to consider the error codes we may wish to respond with.
+> PGDATABASE=nc_games
 
-**Please ensure you work through the tickets in numerical order.**
+Once these files have been created, make sure that they are not being commited to the overall project by checking the the .gitignore and ensuring that the .env.\* is present in that file.
 
-## Git Branching and Pull Requests
+## Running the Scripts
 
-You will be working on each ticket on a new **branch**.
+The next part of the installation is to run `npm init` to install all of the packages in the `package.json`. You'll all need to run `npm i express` and `npm i -D supertest` to ensure that all the files have the appropriate modules to run.
```

Comment:
Do i need to install express and supertest separately if they will be included in the package.json anyway?